integration_test/agent_metrics/metadata.yaml

```diff
@@ -0,0 +1,260 @@
+  labels:
+    device: .*
+    direction: tx|rx
+- type: agent.googleapis.com/interface/packets
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    device: .*
+    direction: tx|rx
+- type: agent.googleapis.com/interface/traffic
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    device: .*
+    direction: tx|rx
+- type: agent.googleapis.com/memory/bytes_used
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    state: buffered|cached|free|slab|used
+- type: agent.googleapis.com/memory/percent_used
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    state: buffered|cached|free|slab|used
+- type: agent.googleapis.com/network/tcp_connections
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    port: all
+    tcp_state: CLOSE|CLOSED|CLOSE_WAIT|CLOSING|DELETE|ESTABLISHED|FIN_WAIT_1|FIN_WAIT1|FIN_WAIT_2|FIN_WAIT2|LAST_ACK|LISTEN|SYN_RECEIVED|SYN_RECV|SYN_SENT|TIME_WAIT
+- type: agent.googleapis.com/pagefile/percent_used
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    state: free|used
+  platform: windows
+- type: agent.googleapis.com/processes/count_by_state
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    state: blocked|daemon|detached|idle|locked|orphan|paging|running|sleeping|stopped|system|unknown|zombies
+  platform: linux
+- type: agent.googleapis.com/processes/cpu_time
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    command: .*
+    command_line: .*
+    owner: .*
+    pid: \d+
+    process: all
+    user_or_syst: user|syst
+- type: agent.googleapis.com/processes/disk/read_bytes_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    command: .*
+    command_line: .*
+    owner: .*
+    pid: \d+
+    process: all
+- type: agent.googleapis.com/processes/disk/write_bytes_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    command: .*
+    command_line: .*
+    owner: .*
+    pid: \d+
+    process: all
+- type: agent.googleapis.com/processes/fork_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+- type: agent.googleapis.com/processes/rss_usage
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    command: .*
+    command_line: .*
+    owner: .*
+    pid: \d+
+    process: all
+- type: agent.googleapis.com/processes/vm_usage
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    command: .*
+    command_line: .*
+    owner: .*
+    pid: \d+
+    process: all
+- type: agent.googleapis.com/swap/bytes_used
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    device: .*
+    state: used|free|cached
```

Comment:
Not really a big deal for regexps, but for human readability, should these be alphabetized? Same for `agent.googleapis.com/swap/percent_used`.`state`.

Reply:
Changed in: [cf8f612](https://github.com/GoogleCloudPlatform/ops-agent/pull/719/commits/cf8f61234082bc521e24290d1f64d70b9a784454)